scipy/stats/morestats.py

```diff
@@ -3478,3 +3478,73 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         R = np.minimum(1, hypot(sin_mean, cos_mean))
 
     return ((high - low)/2.0/pi) * sqrt(-2*log(R))
+
+
+def quartile_coeff_dispersion(samples, q=(0.25, 0.75), axis=0,
+                              interpolation='lower'):
+    """
+    Compute the quartile coefficient of dispersion for a given sample array.
+
+    Given two quartiles Q_low, Q_high from array samples such that
+    Q_low < Q_high, the quartile coefficient of dispersion is calculated by
+    (Q_high - Q_low)/(Q_high + Q_low).
+    by default it is computed using 3rd and 1st quartiles.
+
+    Parameters
+    ----------
+    samples : array_like
+        Input array.
+    q : array_like of float, optional
+        Sequence of 2 quartiles,
+        The default is to compute with ￿Q3 = 0.75, Q1 = 0.25.
```

Comment:
It looks like you tried to fix some of these things, but maybe the changes were partially reversed in a subsequent commit? I'm still seeing the special character here:
![image](https://user-images.githubusercontent.com/6570539/106503781-f25e8300-647a-11eb-8b8e-d5451a7ad26e.png)

 and Q3/Q1 instead of Q_high/Q_low.  Please take a look at the previous comments and check that they were addressed as you intended. Let me know when you'd like me to take another look!

Reply:
@mdhaber, I tried to address every issue. still struggling with my failing CI tests regarding the refguide.